service/start.go

```diff
@@ -18,9 +19,17 @@ func (service *Service) Start() (dockerServices []*swarm.Service, err error) {
 	}
 	// If there is one but not all services running stop to restart all
 	if service.IsPartiallyRunning() {
-		service.Stop()
+		for name, dependency := range service.GetDependencies() {
```

Comment:
I had to change a little bit the logic here in order for the test `TestPartiallyRunningService` to pass. Otherwise the network was destroyed and not re-created.

Reply:
Yes just tried with your modification and without and the tests fails in both cases. The problem doesn't come from here and the `service.Stop()` should be more than enough in that case

Reply:
for some obscure reasons I restarted my docker and now the tests are ok with or without your modification. The docker api was "creating" the network with the same ID but when creating the service with this networkID I got an error that the network was not found and I had the network in my list of network in docker with `docker network list` but even a `docker rm ID` returned me an error that the network didn't exists even it was still on the list.

I guess there is a bug in docker about that but so this modification is not necessary and add complexity so I remove it

Reply:
This fix was only for the CI tests. Locally, both cases also work.